packages/@react-stately/tooltip/test/TooltipManager.test.js

```diff
@@ -0,0 +1,137 @@
+    let tooltipManager = new TooltipManager();
+    let setOpenSpy = jest.fn();
+    let tooltip = {open: false, setOpen: setOpenSpy, tooltipManager};
+    let triggerId = 'triggerId-1';
+
+    tooltipManager.showTooltipDelayed(tooltip, triggerId);
+    expect(setTimeout).toHaveBeenCalledWith(expect.any(Function), 200);
+    jest.runAllTimers();
+
+    let setOpenSpy2 = jest.fn();
+    let tooltip2 = {open: false, setOpen: setOpenSpy2, tooltipManager};
+    let triggerId2 = 'triggerId-2';
+
+    tooltipManager.showTooltipDelayed(tooltip2, triggerId2);
+    expect(setTimeout).toHaveBeenCalledWith(expect.any(Function), 200);
+    jest.runAllTimers();
+
+    expect(setOpenSpy).toHaveBeenLastCalledWith(false);
+    expect(setOpenSpy2).toHaveBeenCalledWith(true);
+    expect(tooltipManager.visibleTooltips).toStrictEqual({triggerId: triggerId2, state: tooltip2});
+    expect(tooltipManager.hoverHideTimeout).toBeNull();
+    expect(tooltipManager.hoverShowTimeout).toBeNull();
+  });
+
+  it('will not show the tooltip if hidden before the delayed show completes', () => {
+    jest.useFakeTimers();
+    let tooltipManager = new TooltipManager();
+    let setOpenSpy = jest.fn();
+    let tooltip = {open: false, setOpen: setOpenSpy, tooltipManager};
+    let triggerId = 'triggerId-1';
+
+    tooltipManager.showTooltipDelayed(tooltip, triggerId);
+
+    expect(tooltipManager.hoverShowTimeout).not.toBeNull();
+    expect(tooltipManager.hoverHideTimeout).toBeNull();
+
+    jest.advanceTimersByTime(100);
+    expect(tooltipManager.visibleTooltips).toBeNull();
+    expect(tooltipManager.hoverShowTimeout).not.toBeNull();
+    expect(tooltipManager.hoverHideTimeout).toBeNull();
+
+    tooltipManager.hideTooltip(tooltip);
+
+    expect(setOpenSpy).toHaveBeenCalledWith(false);
+    expect(tooltipManager.visibleTooltips).toBeNull();
+  });
+
+  it('will not show the first delayed tooltip if a second is delay shown before the first shows', () => {
+    jest.useFakeTimers();
+    let tooltipManager = new TooltipManager();
+    let setOpenSpy = jest.fn();
+    let tooltip = {open: false, setOpen: setOpenSpy, tooltipManager};
+    let triggerId = 'triggerId-1';
+
+    tooltipManager.showTooltipDelayed(tooltip, triggerId);
+    expect(tooltipManager.hoverShowTimeout).not.toBeNull();
+    expect(tooltipManager.hoverHideTimeout).toBeNull();
+
+    jest.advanceTimersByTime(100);
+    expect(tooltipManager.visibleTooltips).toBeNull();
+    expect(tooltipManager.hoverShowTimeout).not.toBeNull();
+    expect(tooltipManager.hoverHideTimeout).toBeNull();
+
+    let setOpenSpy2 = jest.fn();
+    let tooltip2 = {open: false, setOpen: setOpenSpy2, tooltipManager};
+    let triggerId2 = 'triggerId-2';
+
+    tooltipManager.showTooltipDelayed(tooltip2, triggerId2);
+
+    expect(tooltipManager.hoverShowTimeout).not.toBeNull();
+    expect(tooltipManager.hoverHideTimeout).toBeNull();
+
+    // run past first tooltips show timer
+    jest.advanceTimersByTime(150);
```

Comment:
i think there should be a check for something right after this line?
originally I'd checked 
    `expect(tooltipManager.visibleTooltips).toBeNull();`
is there a reason for not checking this?

Reply:
added 👍

Reply:
I don't see the line added?